GetIntoTeachingApi/Program.cs

```diff
@@ -22,6 +25,12 @@ public static async Task Main(string[] args)
                     webBuilder.UseSentry();
                     webBuilder.UseKestrel(opts => opts.AddServerHeader = false);
                     webBuilder.UseStartup<Startup>();
-                });
+                })
+            .UseSerilog(new LoggerConfiguration()
+                .MinimumLevel.Information()
+                .MinimumLevel.Override("Microsoft", LogEventLevel.Warning)
+                .WriteTo.Console()
+                .WriteTo.TCPSink("") // Logstash TCP URI needed here

```

Comment:
I don't think this is needed? I believe `WriteTo.Console` will go to the STDOUT of the Docker container, which is what's piped through to our logstash

Reply:
Ahh, okay!